packages/api/config/ci.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  PORT: 5000,
+  EMAIL_VERIFICATION_TOKEN_EXPIRY: 15 * 60, // 15 minutes
+  LOGIN_TOKEN_EXPIRY: '28d',
+  LOGIN_COOKIE_EXPIRY: 4 * 7 * 24 * 60 * 60 * 1000, // 4 weeks
```

Comment:
Ok. I see that you have `ci.js`, `dev.js` and `prod.js` with repeated key-val pairs.
I think you can do this - 
```js
const commonConfig = { ... };

const ci = {
  ...commonConfig,
  EMAIL_VERIFICATION_TOKEN_EXPIRY: over-ridden value
```

It's not good for a single value to live in 3 files. 

What do you say?

Reply:
That's true. Modified the code now.